client/model.go

```diff
@@ -40,6 +40,73 @@ type ProjectCreatePayload struct {
 	Description string `json:"description"`
 }
 
+type EnvironmentCreate struct {
```

Comment:
what about the TTL here?

Reply:
gonna be added [here](https://trello.com/c/K68z3UXy/4784-terraform-provider-environment-resource-part-2)